docs/source/upgrade_to_newest_version.md

```diff
@@ -0,0 +1,114 @@
+
+At this point, you have two options:
+
+1. If the entire channel is ready to upgrade chaincode, you can upgrade the chaincode on all peers and on the channel (using either the old or new lifecycle depending on the `Application` capability level you have enabled). The best practice at this point would be to vendor the new Go chaincode shim using modules.
+
+2. If the entire channel is not yet ready to upgrade the chaincode, you can use peer environment variables to specify the v1.4 chaincode environment `ccenv` be used to rebuild the chaincode images. This v1.4 `ccenv` should still work with a v2.0 peer.
+
+## Chaincode logger (Go chaincode only)
+
+Support for user chaincodes to utilize the chaincode shim's logger via `NewLogger()`. Chaincodes that used the shim's `NewLogger()` must now shift to their own preferred logging mechanism.
+
+For more information, check out [Logging control](./logging-control.html#chaincode).
+
+## Peer databases upgrade
+
+The databases of all peers (which include not just the state database but the history database and other internal databases for the peer) must be rebuilt using the v2.0 data format as part of the upgrade to v2.0. To trigger the rebuild, the databases must be dropped before the peer is started. To do this, you must first export the `FABRIC_CFG_PATH` to point to the `core.yaml` file for the peer. Then issue:
+
+```
+peer node upgrade-dbs
+```
+
+Once that has completed, issue the following command to start the node:
+
+```
+peer node start
+```
+
+Because rebuilding the databases can be a lengthy process (several hours, depending on the size of your databases), monitor the peer logs to check the status of the rebuild. Every 1000th block you will see a message like `[lockbasedtxmgr] CommitLostBlock -> INFO 041 Recommitting block [1000] to state database` indicating the rebuild is ongoing.
+
+If the database is not dropped as part of the upgrade process, the peer start will return an error message stating that its databases are in the old format and must be dropped. The node will then need to be restarted.
+
+## Capabilities
+
+As can be expected for a 2.0 release, there is a full complement of new capabilities for 2.0.
+
+* **Application** `V2_0`: enables the new chaincode lifecycle as described in [Chaincode for Operators](./chaincode4noah.html).
+
+* **Channel** `V2_0`: requires orderer endpoints to be configured at the org level. For more information, see the "Define orderer endpoint per org" section below.
+
+* **Orderer** `V2_0`: controls `UseChannelCreationPolicyAsAdmins`, changing the way that channel creation transactions are validated. When combined with the `-baseProfile` option of configtxgen, values which were previously inherited from the orderer system channel may now be overridden.
+
+As with any update of the capability levels, make sure to upgrade your peer and ordering node binaries before attempting to update the capability level of any channels.
```

Comment:
As with any update of the capability levels, make sure to upgrade your peer binaries before updating the Application and Channel capabilities, and make sure to upgrade your orderer binaries before updating the Orderer and Channel capabilities.